build-system/test-configs/forbidden-terms.js

```diff
@@ -148,11 +148,9 @@ const forbiddenTermsGlobal = {
   },
   '(?:var|let|const) +IS_FORTESTING +=': {
     message: 'IS_FORTESTING local var only allowed in mode.js.',
-    allowlist: ['src/core/mode/for-testing.js'],
```

Comment:
Why does this allowlist entry get removed?

Reply:
I don't know the syntax for how the forbidden rules work 🤷 . But the rule was no longer activated and so the checker asked that I removed it.  My guess is that since we no longer have a declaration for it, eslint doesn't think its the same var

Reply:
Updated the rules to be much more restrictive